--- usb.ids.converted	2025-10-02 08:56:41.322770441 +0200
+++ usb.ids.updated	2025-10-02 08:56:41.326770423 +0200
@@ -24538,7 +24538,7 @@
 	031  \ and | (Backslash and Bar)
 	032  # and ~ (Hash and Tilde, Non-US Keyboard near right shift)
 	033  ; and : (Semicolon and Colon)
-	034  ´ and " (Accent Acute and Double Quotes)
+	034  Â´ and " (Accent Acute and Double Quotes)
 	035  ` and ~ (Accent Grace and Tilde)
 	036  , and < (Comma and Less)
 	037  . and > (Period and Greater)
