--- hwdata-0.204/MonitorsDB.orig	2007-06-25 06:29:35.000000000 -0500
+++ hwdata-0.204/MonitorsDB	2007-07-11 16:54:38.000000000 -0500
@@ -1164,6 +1164,8 @@
 Dell; Dell 1707FP (Digital); DEL4013; 30.0-81.0; 56.0-76.0; 1
 Dell; Dell 1707FPV(Analog); DEL4021; 30.0-81.0; 56.0-76.0; 1
 Dell; Dell 1707FPV(Digital); DEL4022; 30.0-81.0; 56.0-76.0; 1
+Dell; Dell 1708FP(Analog); DEL4023; 30.0-81.0; 56.0-76.0; 1
+Dell; Dell 1708FP(Digital); DEL4024; 30.0-81.0; 56.0-76.0; 1
 Dell; Dell 1726T-HS/D1025HT; del5319; 31.0-85.0; 50.0-100.0; 1
 Dell; Dell 1800FP (Analog); DELE000; 30.0-80.0; 56.0-85.0; 1
 Dell; Dell 1800FP (Analog); DELE002; 30.0-80.0; 56.0-75.0; 1
@@ -1182,6 +1184,8 @@
 Dell; Dell 1907FP (Analog); DEL4014; 30.0-81.0; 56.0-76.0; 1
 Dell; Dell 1907FP (Digital); DEL4015; 30.0-81.0; 56.0-76.0; 1
 Dell; Dell 1907FPV(Analog); DEL4019; 30.0-81.0; 56.0-76.0; 1
+Dell; Dell 1908FP(Analog); DEL4025; 30.0-81.0; 56.0-76.0; 1
+Dell; Dell 1908FP(Digital); DEL4026; 30.0-81.0; 56.0-76.0; 1
 Dell; Dell 1907FPV(Digital); DEL4020; 30.0-81.0; 56.0-76.0; 1
 Dell; Dell 2000FP (Analog); DELA002; 31.5-80.0; 56.0-76.0; 1
 Dell; Dell 2000FP (Digital); DELA003; 31.5-80.0; 56.0-76.0; 1
@@ -1229,9 +1233,11 @@
 Dell; Dell E173FP; DELA00B; 31.0-80.0; 56.0-76.0; 1
 Dell; Dell E176FP; DELA014; 31.0-80.0; 56.0-75.0; 1
 Dell; Dell E177FP; DELA023; 31.0-80.0; 56.0-75.0; 1
+Dell; Dell E178FP; DELA027; 31.0-80.0; 56.0-75.0; 1
 Dell; Dell E193FP; DEL700E; 31.0-83.0; 56.0-76.0; 1
 Dell; Dell E196FP; DELA015; 31.0-83.0; 56.0-76.0; 1
 Dell; Dell E197FP; DELA024; 31.0-83.0; 56.0-76.0; 1
+Dell; Dell E198FP; DELA028; 31.0-83.0; 56.0-76.0; 1
 Dell; DELL E207WFP; DELD010; 30.0-83.0; 56.0-75.0; 1
 Dell; Dell E207WFP; DELD011; 30.0-83.0; 56.0-75.0; 1
 Dell; Dell E228WFP; DELD015; 30.0-83.0; 56.0-75.0; 1
@@ -1286,6 +1292,8 @@
 Dell; Dell P992; DEL5002; 30.0-107.0; 48.0-170.0; 1
 Dell; Dell SE177FP; DELF001; 30.0-81.0; 56.0-76.0; 1
 Dell; Dell SE197FP; DELF002; 30.0-81.0; 56.0-76.0; 1
+Dell; Dell SP1908FP; DEL4030; 30.0-81.0; 56.0-76.0; 1
+Dell; Dell SP1908FP (DVI); DEL4031; 30.0-81.0; 56.0-76.0; 1
 Dell; Dell Super VGA; 0; 30.0-38.5; 50.0-90.0; 1
 Dell; Dell Super VGA Colour; 0; 29.0-38.0; 47.0-100.0; 1
 Dell; Dell Super VGA DL 1428 I/L; 0; 35.5; 87.0; 1
