--- pnp.ids.orig	2021-04-01 14:18:27.034313311 +0200
+++ pnp.ids	2021-04-01 14:20:19.923035707 +0200
@@ -191,7 +191,7 @@
 ATX	Athenix Corporation
 AUG	August Home, Inc.
 AUI	ALPS ALPINE CO., LTD.
-AUO	DO NOT USE - AUO
+AUO	AU Optronics
 AUR	Aureal Semiconductor
 AUS	ASUSTek COMPUTER INC
 AUT	Autotime Corporation
@@ -543,7 +543,7 @@
 DIN	Daintelecom Co., Ltd
 DIS	Diseda S.A.
 DIT	Dragon Information Technology
-DJE	Capstone Visua lProduct Development
+DJE	Capstone Visual Product Development
 DJP	Maygay Machines, Ltd
 DKY	Datakey Inc
 DLB	Dolby Laboratories Inc.
@@ -865,7 +865,7 @@
 GRY	Robert Gray Company
 GSB	NIPPONDENCHI CO,.LTD
 GSC	General Standards Corporation
-GSM	Goldstar Company Ltd
+GSM	LG Electronics
 GSN	Grandstream Networks, Inc.
 GST	Graphic SystemTechnology
 GSY	Grossenbacher Systeme AG
@@ -1250,7 +1250,7 @@
 LOL	Litelogic Operations Ltd
 LPE	El-PUSK Co., Ltd.
 LPI	Design Technology
-LPL	DO NOT USE - LPL
+LPL	LG Philips
 LSC	LifeSize Communications
 LSD	Intersil Corporation
 LSI	Loughborough Sound Images
@@ -2265,9 +2265,9 @@
 UNA	Unisys DSD
 UNB	Unisys Corporation
 UNC	Unisys Corporation
-UND	DO NOT USE - UND
-UNE	DO NOT USE - UNE
-UNF	DO NOT USE - UNF
+UND	Unisys Corporation
+UNE	Unisys Corporation
+UNF	Unisys Corporation
 UNI	Uniform Industry Corp.
 UNM	Unisys Corporation
 UNO	Unisys Corporation
@@ -2403,7 +2403,7 @@
 WWV	World Wide Video, Inc.
 WXT	Woxter Technology Co. Ltd
 WYR	WyreStorm Technologies LLC
-WYS	Myse Technology
+WYS	Wyse Technology
 WYT	Wooyoung Image & Information Co.,Ltd.
 XAC	XAC Automation Corp
 XAD	Alpha Data
